### ISSUE TITLE
Syntax Error by Following Documentation Code

### ISSUE THREAD

**Issue by pkqxdd** [mitmproxy/mitmproxy#3255]
I followed the "workaround" for macOS for transparent proxying. However, after I pasted the code, pfctl gave the following output: 
```
No ALTQ support in kernel
ALTQ related functions disabled
pf.conf:23: syntax error
pfctl: Syntax error in config file: pf rules not loaded
```
##### System information
Mitmproxy: 3.0.4
Python:    3.6.6
OpenSSL:   OpenSSL 1.1.0h  27 Mar 2018
Platform:  Darwin-17.6.0-x86_64-i386-64bit
SIP disabled

**Comment by Kriechi** [comment]
What exactly did you paste?
There recently was a PR that added a keyword to the pf config: #3239
Does this fix your issue?

**Comment by pkqxdd** [comment]
I used this example code 
```
#The ports to redirect to proxy
redir_ports = "{http, https}"

#The address the transparent proxy is listening on
tproxy = "127.0.0.1 port 8080"

#The user the transparent proxy is running as
tproxy_user = "nobody"

#The users whose connection must be redirected.
#
#This cannot involve the user which runs the
#transparent proxy as that would cause an infinite loop.
#
#Here we redirect for all users which don't run transparent proxy.
redir_users = "{ !=" $tproxy_user "}"

#If you only wish to redirect traffic for particular users
#you may also do:
#redir_users = "{= john, = jane}"

rdr pass proto tcp from any to any port $redir_ports -> $tproxy
pass out route-to (lo0 127.0.0.1) proto tcp from any to any port $redir_ports user $redir_users
```
The PR did not change it. The syntax error seems to occur in the last line of this snippet.

**Comment by akx** [comment]
Seeing the same problem here.

**Comment by king6cong** [comment]
same problem here

**Comment by king6cong** [comment]
change line 23 to 
```
pass out route-to (lo0 127.0.0.1) proto tcp from any to any port $redir_ports user { != nobody }
```

solves the problem

**Comment by Kriechi** [comment]
Fixed by #3600
